perf/lua/1mops_write.lua

```diff
@@ -192,6 +192,12 @@ if (nodes > 1) then
         for i = 3302, 3300+nodes do
             -- subdir for replica's data
             os.execute('mkdir ' .. i)
+            local uri
+            if type(box.info.listen) == 'table' then
+                uri = box.info.listen[1]
```

Comment:
Wouldn't it be better to bind to one interface instead?

```
box.cfg{listen = '127.0.0.1:0'}
```

Reply:
Fixed.

```diff
 box.cfg{
     log_level = 0,
-    listen = 0,
+    listen = '127.0.0.0:0',
     read_only = false,
```